build-utils/rebuild_backend_images.sh

```diff
@@ -5,6 +5,8 @@ gradle_dependencies=$(grep gradle_dependencies ./build-utils/common.conf | cut -
 dependencies="./dataland-backend/ ./dataland-backend-utils/ ./dataland-keycloak-adapter/ ./dataland-message-queue-utils/ $gradle_dependencies"
```

Comment:
In the last ticket, I created the data-upload-process documentation. I think it would be a good idea to create a similar documentation for datapoints and (hopefully) keep it up-to-date for as long as possible. What do you think?

Reply:
I adapted the old documentation and added a new one for data points